core/storage/base_model/gae_models.py

```diff
@@ -1084,10 +1084,10 @@ def get_version(cls, entity_id, version_number, strict=True):
                 id=entity_id,
                 version=version_number
             )._reconstitute_from_snapshot_id(snapshot_id)
-        except cls.EntityNotFoundError as e:
+        except cls.EntityNotFoundError:
             if not strict:
                 return None
-            raise e
+            python_utils.reraise_exception()
```

Comment:
The exception won't have the exception message?

Reply:
It will and it will also contain a proper stack trace (the exception message is retrieved with `sys` in `reraise_exception`).